src/routes/SignupPage.tsx

```diff
@@ -571,6 +582,18 @@ const SignupPage = () => {
                 I'm not sure
               </label>
             </div>
+
+            <div className="editprofile-form-input" >
+                <label htmlFor="editprofile-radius" >Radius
+                    <span className="tooltip">
+                        <i className="fa-solid fa-circle-info"></i>
+                        <span className="tooltiptext">The default search area for exploring nearby rides. ChainLink will default to this radius when display upcoming rides on your explore page.</span>
```

Comment:
This should match the tooltip on the other screen

Reply:
wait, I'm a clutz, need to remove something else